modules/default/calendar/calendar.js

```diff
@@ -337,6 +337,9 @@ Module.register("calendar", {
 							}
 							break;
 						}
+						if (typeof this.config.customEvents[ev].eventClass !== "undefined" && this.config.customEvents[ev].eventClass !== "") {
```

Comment:
you should add info about eventClass to line 44 (and later in a seperate PR to the documentation repo)

Reply:
Done ;-)